scripts/gh_pr.sh

```
@@ -0,0 +1,149 @@
+
+    # Fetch PRs and their statuses in parallel-ish or sequential for simplicity
+    local prs=$(curl -s "$API_URL/pulls")
+
+    echo "$prs" | jq -r '.[] | "\(.number)\t\(.user.login)\t\(.title)"' | while read -r line; do
+        id=$(echo "$line" | cut -f1)
+        user=$(echo "$line" | cut -f2)
+        title=$(echo "$line" | cut -f3)
+
+        # Get CI status (simplified to last check-run)
+        local status_json=$(curl -s "$API_URL/commits/pull/$id/head/check-runs")
+        local conclusion=$(echo "$status_json" | jq -r '.check_runs[0].conclusion // "pending"')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)